nip47/publish_nip47_info.go

```diff
@@ -16,6 +16,35 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+type Nip47InfoPublishRequest struct {
+	AppWalletPubKey  string
+	AppWalletPrivKey string
+}
+
+type nip47InfoPublishQueue struct {
+	channel chan *Nip47InfoPublishRequest
+}
+
+func NewNip47InfoPublishQueue() *nip47InfoPublishQueue {
+	return &nip47InfoPublishQueue{
+		channel: make(chan *Nip47InfoPublishRequest, 1000),
```

Comment:
This might become an issue if a hub has more than 1000 apps. I guess we log an error but I wonder if we need to be explicit here with the number? (I am just thinking about e.g. Yakihonne or https://lnwallet.app/ which allow creating new wallets with one click)

Reply:
Removed size restriction :+1:

Reply:
@im-adithya this was completely wrong, please test your changes. I've fixed it in the latest commit

Reply:
Sorry for that, should've tested after changing!